.github/workflows/test.yaml

```diff
@@ -191,9 +214,35 @@ jobs:
         git reset --hard $BRANCH_SHA
 
     - name: Newsfragment
+      if: ${{ !cancelled() }}
```

Comment:
I don't know why this is here.  Maybe a comment would help.

Reply:
OK. I will add a comment.

This is here so that newsfragment check will run, even if the previous fails.

Reply:
In the end I decided to just revert this change.

The plan is to upgrade to the latest version of towncrier , and then we will refactor this to run in a separate job.